src/Microsoft.ML.Data/Transforms/SkipTakeFilter.cs

```diff
@@ -270,4 +270,30 @@ protected override bool MoveManyCore(long count)
             }
         }
     }
+
+    public static class SkipFilter
+    {
+        /// <summary>
+        /// A helper method to create'SkipFilter' transform by skipping the number of rows defined by the <paramref name="count"/> parameter.
```

Comment:
>SkipFilter [](start = 38, length = 10)

There is no such thing as a `SkipFilter` transform per se. Let's just clarify that this is a `SkipTakeFilter` (since it is), and use a proper `<see` tag of course.